src/index.d.ts

```diff
@@ -96,10 +109,10 @@ declare namespace SDKStandardComponents {
             credentialType: 'FIDO';
```

Comment:
I've seen TCredential type used here in your other PR ;-)

Reply:
I typically dislike branching off other branches. The plan is to rebase the changes. 
#75 > #76 > #77 > #78